indico/web/client/js/react/components/index.js

```diff
@@ -22,6 +22,7 @@ export {default as TooltipIfTruncated} from './TooltipIfTruncated';
 export {default as SingleDatePicker, FinalSingleDatePicker} from './dates/SingleDatePicker';
 export {default as DatePicker, FinalDatePicker} from './DatePicker';
 export {default as DateRangePicker} from './dates/DateRangePicker';
+export {default as DateRangePicker2} from './DateRangePicker';
```

Comment:
I hope we can replace the `DateRangePicker` with the new one and thus not need to keep the old one around :)